web/src/app.svelte

```diff
@@ -54,9 +54,7 @@
   {#if !$captureMode}
     <div class="flex justify-between px-2 sm:px-4 py-2 items-center">
       <div>
-        <h1 class="text-2xl sm:text-3xl text-sky-500 dark:text-sky-400">
-          {'<' + import.meta.env.VITE_SITE_NAME + ' />'}
-        </h1>
```

Comment:
This project also hosts the Indonesia website at https://id.techcal.dev/

When we merge this, the header for that website will also change on that website.

Can you make it so that it displays the new logo only on Thai site, and display the old text on Indonesia site? You can use `{#if }` ... `{:else}` ... `{/if}` block to check if `import.meta.env.VITE_SITE_NAME === 'thai-tech-calendar'`